src/plone/restapi/testing.zcml

```diff
@@ -5,6 +5,11 @@
   xmlns:zcml="http://namespaces.zope.org/zcml"
   i18n_domain="plone.app.contenttypes">
 
+  <utility
```

Comment:
@csenger is it possible to create this vocabulary outside of the test profile just for the single test. I somehow dislike testing profiles and global test setups. I would like to get of rid of this testing profile at some point if possible.